src/helper/remote_config/service_handler.hpp

```diff
@@ -0,0 +1,68 @@
+// Unless explicitly stated otherwise all files in this repository are
+// dual-licensed under the Apache-2.0 License or BSD-3-Clause License.
+//
+// This product includes software developed at Datadog
+// (https://www.datadoghq.com/). Copyright 2021 Datadog, Inc.
+#pragma once
+
+#include "engine.hpp"
+#include "remote_config/client.hpp"
+#include "remote_config/settings.hpp"
+#include "service_config.hpp"
+#include "service_identifier.hpp"
+#include "std_logging.hpp"
+#include "utils.hpp"
+#include <future>
+#include <memory>
+#include <spdlog/spdlog.h>
+#include <unordered_map>
+
+namespace dds::remote_config {
+
+using namespace std::chrono_literals;
+
+class service_handler {
```

Comment:
I wonder if `client_handler` is more accurate.

Reply:
We chat about this and said either `service_handler` or `client_handler` would be good. I will change it anyways...